project/Settings.scala

```diff
@@ -135,6 +135,18 @@ object Settings {
     libraryDependencies ++= databaseMigrationDependencies
   ) ++ commonSettings
 
+  val cromwellApiClientSettings = List(
+    name := "cromwell-api-client",
+    libraryDependencies ++= cromwellApiClientDependencies,
+    organization := "org.broadinstitute",
+    scalaVersion := "2.12.1",
```

Comment:
Attempted here: https://github.com/broadinstitute/cromwell/compare/cjl_cromwell_api_client...cjl_cromwell_api_client_crosspub?expand=1

The cross-publishing was a bit of a mess with dependencies' versions changing for 2.12. Let's tech talk once I'm back Wednesday.